code/datums/emergency_calls/emergency_call.dm

```diff
@@ -235,25 +251,13 @@
 
 
 /datum/emergency_call/proc/get_spawn_point(is_for_items)
-	var/index
+	var/obj/effect/landmark/L
 	if(is_for_items)
-		index = "[name_of_spawn]Item"
+		L = pick(shuttle?.item_spawns)
 	else
-		index = name_of_spawn
-	if(!GLOB.distress_spawns_by_name[index])
-		return FALSE
-
-	var/list/spawn_list = GLOB.distress_spawns_by_name[index].Copy()
-
-	if(!length(spawn_list)) //Empty list somehow
-		return FALSE
-
-	var/turf/spawn_loc	= pick(spawn_list)
-	if(!istype(spawn_loc))
-		return FALSE
-
-	return spawn_loc
-
+		L = pick(shuttle?.mob_spawns)
+	if(L)
+		return get_turf(L)
```

Comment:
```suggestion
		return L ? get_turf(L) : FALSE
```
And remove the `if` check?

Reply:
less readable

Reply:
Do an early return then. Make it clear what's being returned.

Reply:
disagree

Reply:
it'll return null which is fine

Reply:
Could you make that more explicit?